src/Parser/Rogue/Assassination/Modules/Features/Abilities.js

```diff
@@ -17,7 +17,7 @@ class Abilities extends CoreAbilities {
         castEfficiency: {
           suggestion: true,
           recommendedEfficiency: 1.0,
-          extraSuggestion: 'In most cases should be used on cooldown',
+          extraSuggestion: 'Only delay Vendetta if it will not reduce the total amount of casts.',
```

Comment:
I would just remove the extraSuggestion here if you don't want to go into specifics.  Toxic Blade and Kingsbane have good extraSuggestions because they go into specifics on when you hold the ability or why you want to make sure to use it on CD.  Here and in DFA the extra suggestion doesn't add anything useful for the player.  We want to avoid redundant text and make text important so that players don't start skimming the text and potentially missing something important.

Reply:
ok, removed